include/clover_field_order.h

```diff
@@ -500,10 +496,11 @@ namespace quda {
          * @param[in] dim Which dimension we are taking the minimum abs of (dummy for clover)
          * @return Minimum norm
          */
-        __host__ double abs_min(int dim=-1, bool global=true) const {
-          double absmax = accessor.transform_reduce(location, abs_<Float, Float>(), std::numeric_limits<double>::max(),
+        __host__ double abs_min(int =-1, bool global=true) const {
+          commGlobalReductionPush(global);
+          double absmax = accessor.transform_reduce(location, abs_min_<Float, Float>(), std::numeric_limits<double>::max(),
```

Comment:
Cleanup since it's quick + while we're here, `double absmax` -> `double absmin`

Reply:
d45340b861555b1885fb7d3c5699db686fd6ac77